sklearn/utils/tests/test_extmath.py

```diff
@@ -49,6 +49,19 @@ def test_density():
         assert density(X_) == density(X)
 
 
+def test_density_deprecated_kwargs():
```

Comment:
Let's put a reminder here for when we need to complete the deprecation in version `1.4`. This makes it more easily searchable 
```suggestion
# TODO(1.4): Remove test
def test_density_deprecated_kwargs():
```